_posts/2018-01-22-portus-2.3.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Portus 2.3 released!
+author: Miquel Sabaté Solà
+layout: blogpost
+---
+
+It's been almost a year since we released Portus 2.2, but we can finally present
+to you the latest release of Portus. We wanted to deliver this new release much
+sooner, but some late-coming features changed the release schedule quite a bit,
+so we had to spend more time for the final release.
+
+For this release we have improved lots of small UX details and we have fixed as
+many bugs as we could from the 2.2 release. You can check the
+[changelog](https://github.com/SUSE/Portus/blob/master/CHANGELOG.md) for more
+info of this release, but in this blog post we want to highlight the following
+new features:
+
+#### Security scanning
+
+Portus is now able to scan security vulnerabilities on your Docker images. This
+is done with different backends, where the stable one is [CoreOS
```

Comment:
`The scanner we tested and recommend is CoreOS Clair but you can also use a different backend for it. You have to enable...`

Rephase to something like that? I dunno...

Reply:
I don't know, the other ones are not usable as of now 😅

Reply:
Then we should not mention `different backends`. It's not sounding good to me but I dunno how to help with that. :(